packages/apollo-tools/src/__tests__/snapshotSerializers/astSerializer.ts

```diff
@@ -12,7 +12,7 @@ export = {
     indentation: string,
     depth: number,
     refs: Refs,
-    printer: Printer
+    printer: any
```

Comment:
Why not use the `Printer` type? was it removed from the exports?

Reply:
I only vaguely recall why I did this, and it has to do with type incompatibility from `pretty-format`'s types =, however it works perfectly fine at runtime. Decided to remove these unused params anyway!